containers/podman-desktop
# feat: add custompick component (#2855)

### What does this PR do?

This PR adds a custompick component so extensions can show a options in a custom dialog.

### Screenshot/screencast of this PR

when having 2 items
![two_items_full_length](https://github.com/containers/podman-desktop/assets/49404737/f431ecdb-55ce-4f44-970f-2f770538ecb3)

with >3
![custompick_draggable](https://github.com/containers/podman-desktop/assets/49404737/23df6d5d-c531-48ef-8460-11822190c915)

### What issues does this PR fix or reference?

it resolves #2855 

### How to test this PR?

1. install quay.io/lstocchi/sample-podman-extension:custompick_v3
2. click on the custompick item in the statusbar
3. select an option and check that it's index is printed in the console


P.S: Adding some tests but it's ready if you want to give a look
